examples/webgl_postprocessing_msaa_unbiased.html

```diff
@@ -223,13 +228,21 @@
 
 				}
 
-				msaaRenderPassP.sampleLevel = param.sampleLevel;
-				msaaRenderPassP.unbiased = param.unbiased;
+				var newColor = msaaRenderPassP.clearColor;
+				switch( params.clearColor ) {
+					case 'blue': newColor = new THREE.Color( 0x0000ff ); break;
+					case 'red': newColor = new THREE.Color( 0xff0000 ); break;
+					case 'green': newColor = new THREE.Color( 0x00ff00 ); break;
+					case 'white': newColor =  new THREE.Color( 0xffffff ); break;
+					case 'black': newColor =  new THREE.Color( 0x000000 ); break;
+				}
+				msaaRenderPassP.clearColor = msaaRenderPassO.clearColor = newColor;
```

Comment:
I know it is minor, but it probably would be good to rewite at some point to avoid instantiating a new `Color` every frame.